src/open_inwoner/core/views.py

```diff
@@ -49,56 +53,122 @@ def sitemap(request):
         ]
 
+        # conditionally add cms pages based on apphook configuration
+        cms_pages = context["cms_profile_pages"]
+        profile_config = ProfileConfig.objects.get()
+        if profile_config.selected_categories:
+            base_url = reverse("profile:contact_list")
+            begeleiders_url = f"{base_url}?{urlencode({'type': 'begeleider'})}"
+            cms_pages.append(
+                {
+                    "url": begeleiders_url,
+                    "link_text": _("Mijn begeleiders"),
+                },
+            )
+        if profile_config.mentors:
+            cms_pages.append(
+                {
+                    "url_name": "profile:contact_list",
+                    "link_text": _("Mijn contacten"),
+                },
+            )
+        if profile_config.my_contacts:
+            cms_pages.append(
+                {
+                    "url_name": "profile:contact_list",
+                    "link_text": _("Mijn contacten"),
+                },
+            )
+        if (
+            profile_config.selfdiagnose
+            and QuestionnaireStep.objects.filter(published=True).exists()
+        ):
+            cms_pages.append(
+                {
+                    "url_name": "products:questionnaire_list",
+                    "link_text": _("Zelfdiagnose"),
+                }
+            )
+        if profile_config.actions:
+            cms_pages.append(
+                {
+                    "url_name": "profile:action_list",
+                    "link_text": _("Openstaande acties"),
+                },
+            )
+        if profile_config.notifications:
+            cms_pages.append(
+                {
+                    "url_name": "profile:notifications",
+                    "link_text": _("Notificatievoorkeuren"),
+                }
+            )
+        if profile_config.questions:
+            cms_pages.append(
+                {
+                    "url_name": "cases:contactmoment_list",
+                    "link_text": _("Mijn vragen"),
+                }
+            )
+        if profile_config.ssd:
+            cms_pages.append(
+                {
+                    "url_name": "ssd:uitkeringen",
+                    "link_text": _("Mijn uitkeringen"),
+                }
+            )
+        if profile_config.appointments:
+            cms_pages.append(
+                {
+                    "url_name": "profile:appointments",
+                    "link_text": _("Mijn afspraken"),
+                },
+            )
+
     # add "platform pages" (conditional login/register page, cookie info etc.)
     context["platform_pages"] = [
-        {"url_name": "pages-root", "link_text": "Home page"},
-        {"url_name": "contactform", "link_text": "Contactformulier"},
+        {"url_name": "pages-root", "link_text": _("Home page")},
     ]
 
+    klant_config = KlantenSysteemConfig.get_solo()
+    if klant_config.has_contactform_configuration:
```

Comment:
Something weird is going on with the general Contactform - it has disappeared from the space where the flat pages used to be so it does show up correctly in the sitemap, however the page itself is now empty,
also you will need to make the "Onderwerpen" heading conditional in case a municipality makes the Category pages invisible for non-authenticated users.

You can see a living example of both issues here:
https://mijn-acc.groningen.nl/sitemap/

Reply:
- The contactform issue should be fixed already (you can try by setting the default Klant backend to null in the KlantSystem configuration); the contactform should disappear from the links.
- Good catch concerning the "Onderwerpen" heading; made it conditional